core/src/AbilityHelpers.ts

```diff
@@ -169,8 +179,19 @@ class AbilityHelpers implements Types.AbilityHelpersCore, Types.AbilityHelpersIn
     }
 }
 
-export function createAbilityHelpers(win: Window, props?: Types.AbilityHelpersCoreProps): Types.AbilityHelpersCore {
-    return new AbilityHelpers(win, props);
+/**
+ * Creates an instance of ability helpers, returns the current window instance if it already exists
+ */
+export function createAbilityHelpers(win: WindowWithAHInstance, props?: Types.AbilityHelpersCoreProps): Types.AbilityHelpersCore {
+    const existingAh = getCurrentAbilityHelpers(win);
+    if (existingAh) {
```

Comment:
Makes sense to add `if (__DEV__)` to the console log. So that it's not affection the production bundle.

Reply:
yeah you're right